Update naming.md to clarify 'duration' definition (#3543)
open-telemetry/semantic-conventions: docs/general/naming.md

Author: Weird I accepted the gh suggestion... Fixing this.

  defined as the difference in `system.cpu.time` measurements divided by the
  elapsed time and number of CPUs.

- **duration** - a histogram that measures operation duration
  should be called `{operation name}.duration`.
  For example, `http.server.request.duration` for the time taken to process each HTTP request.
  The difference with `time` is that `time` is used to measure monotonically increasing total time,

>   whereas `duration` captures the aggregated elapsed time.

Reviewer: the word "aggregated" is a bit confusing here, maybe something like
```suggestion
  whereas `duration` captures the elapsed time of discrete operations
```

>   whereas `duration` captures the elapsed time of discrete operations

Reviewer 2: ```suggestion
  whereas `duration` captures the elapsed time of discrete operations.
```

  defined as the difference in `system.cpu.time` measurements divided by the
  elapsed time and number of CPUs.

- **duration** - an instrument that measures elapsed time for individual occurrences 

>   of an event should be called `entity.duration`.

Reviewer 3: ```suggestion
  of an event should be called `{operation name}.duration`.
```

entity in otel mean something special and would be confusing here, but 'operation name' is used throughout the repo
